augur/tree.py

```diff
@@ -244,17 +246,17 @@ def random_string(n):
     # requesting as many threads as there are sequences when there may be fewer
     # available threads on the current machine.
     if num_seqs < nthreads:
-        nthreads = "AUTO"
+        tree_builder_args = tree_builder_args + " -nt AUTO"
```

Comment:
I haven't tested to be sure, but I think we don't even need this block to check number of sequences vs. number of threads, since we use `-nt AUTO` by default for IQ-TREE now. In this case, adding `-nt AUTO` again would be redundant with the defaults and if someone has overridden the defaults, I'd say they think they know what they're doing and should get an error about number of sequences < number of threads.

Reply:
I agree - I added this just to make the changes as small as possible.

Removing the section as we now use `-nt AUTO` by default gets rid of code which is great!

Reply:
Cool! If you have a chance, can you confirm that IQ-TREE does "the right thing" when you try to build a tree with more max CPUs than available sequences using this new approach? I just want to make sure we don't cause a regression unwittingly.